src/mappings/events/vaults.ts

```diff
@@ -89,8 +93,26 @@ export const vaultsEventKit = (block: any, data: any, module: string, path: stri
       );
     }
 
-    vault.coin = payload?.locked?.__brand;
-    vault.denom = payload?.locked?.__brand;
+    const vaultGovernanceId = id.split('.').slice(0, 4).join('.') + '.governance';
+    const vaultManagerGovernance = await VaultManagerGovernance.get(vaultGovernanceId);
+
+    const oraclPriceId = `${denom}-USD`;
+    const oraclePrice = await OraclePrice.get(oraclPriceId);
+
+    if (vaultManagerGovernance && vault.vaultManagerGovernance === undefined)
+      vault.vaultManagerGovernance = {
```

Comment:
I think we should also handle the case when `vaultManagerGovernance` is an empty object. Perhaps add some fallback values?

Reply:
`vaultManagerGovernace` is not a json object object, but rather a class object. there is no empty state for it